ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -4353,6 +4353,72 @@ tests:
     - ref: cucushift-installer-check-realtime
     - chain: openshift-e2e-test-qe
     workflow: baremetal-lab-upi-dual-stack
+- as: aws-short-cert-rotation-f9
+  cron: 31 13 7,16,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+    test:
+    - ref: openshift-e2e-cert-rotation-observe
+    - ref: openshift-e2e-cert-rotation-test
+    - ref: openshift-extended-test
+    workflow: openshift-e2e-cert-rotation-short
+- as: gcp-short-cert-rotation-f9
```

Comment:
From defination of workflow https://github.com/openshift/release/blob/ac997d550b6fd5a38e9835db2c0808db2afa16c9/ci-operator/step-registry/openshift/e2e/cert-rotation/short/openshift-e2e-cert-rotation-short-workflow.yaml#L6, it's only for `AWS`, I don't think it can work on other platforms.